### PR TITLE
feat: convert schema files to .ts in msft component package

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Closes: #1632

Convert schemas over from .json to .ts files and export them accordingly.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components-react-msft/src/flipper/flipper.schema.json [hunk 1]
```diff
@@ -1,40 +0,0 @@
-{
```
Comment (janechu) [inline packages/fast-components-react-msft/src/flipper/flipper.schema.json]: This file is still being deleted

File: packages/fast-components-react-msft/src/action-toggle/index.ts [hunk 2]
```diff
@@ -33,5 +34,6 @@ export {
     ActionToggleProps,
     ActionToggleClassNameContract,
     ActionToggleHandledProps,
+    actionToggleSchema,
```
Comment (chrisdholt) [inline packages/fast-components-react-msft/src/action-toggle/index.ts]: I think we should have this follow our current syntax for exports: `ActionToggleSchema`

File: packages/fast-components-react-msft/src/action-trigger/index.ts [hunk 3]
```diff
@@ -36,5 +37,6 @@ export {
     ActionTriggerProps,
     ActionTriggerClassNameContract,
     ActionTriggerHandledProps,
+    actionTriggerSchema,
```
Comment (chrisdholt) [inline packages/fast-components-react-msft/src/action-trigger/index.ts]: Same here, and turtles all the way down on naming: 

`ActionTriggerSchema`.

File: packages/fast-components-react-msft/src/action-toggle/index.ts (hunk 2 above)
Comment (eljefe223) [reply packages/fast-components-react-msft/src/action-toggle/index.ts]: Talked offline decided to leave them camel cased

**Pull request merged**